librosa/sequence.py

```diff
@@ -272,12 +272,14 @@ def dtw(
     -------
     D : np.ndarray [shape=(N, M)]
         accumulated cost matrix.
-        D[N, M] is the total alignment cost.
-        When doing subsequence DTW, D[N,:] indicates a matching function.
-    wp : np.ndarray [shape=(N, 2)]
+        The final value ``D[-1, -1]`` is the total alignment cost.
```

Comment:
could you add "at index position" here? My brain was (don't know why) tempted to interpret it as $D \in [-1, 1]$ first.